tests/JsonErrorFactoryTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use WtfPhp\JsonApiErrors\Exceptions\JsonErrorException;
+use WtfPhp\JsonApiErrors\Factories\JsonErrorFactory;
+use WtfPhp\JsonApiErrors\Models\JsonError;
+
+class JsonErrorFactoryTest extends TestCase
+{
+    /** @test */
+    public function itShouldReturnAJsonErrorObject() {
+        $e = new Exception('foo', 200);
+        $this->assertInstanceOf(JsonError::class, JsonErrorFactory::createFromThrowable($e));
```

Comment:
Why not also check here if the `message` and the `code` have been set correctly?

Reply:
I already implemented 2 tests for message and code.
`itShouldContainTheErrorCode` and `itShouldContainTheErrorTitle`